Makefile

```diff
@@ -497,20 +497,28 @@ webhook-certs-dir:
 _ensure-kong-system-namespace:
 	@kubectl create ns kong-system 2>/dev/null || true
 
+TMP_DIR := $(shell mktemp -d)
```

Comment:
I just realized this: this will cause a new temp dir to be created every time `make` is called right? Can we put this into `impersonate-kgo` to prevent this?

Reply:
To solve this we could use something like: https://github.com/Kong/gateway-operator/compare/mlavacca/make-run-impersonate...pmalek/make-run-impersonate?expand=1

Reply:
For posterity, @pmalek implemented this in d246518872ffcb668b40440eee747f031fd24867